oz/__init__.py

```diff
@@ -131,7 +130,18 @@ def get_template_namespace(self):
     def initialize(self, **kwargs):
         self.trigger("initialize", **kwargs)
 
+    def set_secure_cookie(self, name, value, **kwargs):
+        if self.settings["use_secure_cookie"]:
+            if "secure" not in kwargs:
+                kwargs["secure"] = True
+            if "httponly" not in kwargs:
+                kwargs["httponly"] = True
+        super(RequestHandler, self).set_secure_cookie(name, value, **kwargs)
+
     def prepare(self):
+        if self.settings["use_hsts"]:
+            self.set_header("Strict-Transport-Security", "max-age=2592000; includeSubDomains")
```

Comment:
yes, we do. but they'll also get this hsts header before redirecting, right? and the question is, what happens if you get the hsts header when you're not on https? I'm pretty sure it's fine, but checking just in case.